android/res/values-be/strings.xml

```diff
@@ -24,8 +24,10 @@
 	<string name="gb">ГБ</string>
 	<!-- Choose measurement on first launch alert - choose imperial system button -->
 	<string name="miles">Мілі</string>
-	<!-- View and button titles for accessibility -->
+	<!-- A text for current gps location point/arrow selected on the map -->
 	<string name="core_my_position">Маё месцазнаходжанне</string>
+	<!-- A default name for a bookmark created off current gps location, %@ is creation time and date, e.g. "Me at 17:10 19.01.2022" -->
+	<string name="my_position_bookmark_name">Я ў %s</string>
```

Comment:
Proposed data/time style looks very confusing to me:

- "10:28 06.02.2022" (Russian) 
- "10:28 06/02/2022" (English - UK)
- "10:28 2/6/2022" (English - UK)

Please consider using different date/time style. For example, "Sun Feb 6 10:28" is significantly more readable than "10:28 06/02/2022".